lambda/events/pet.json

```diff
@@ -1,5 +1,5 @@
 {
-  "body": "{\"type\":\"cat\",\"image\":null,\"demoSession\":\"test\"}",
```

Comment:
Don't we now need to have `create-pet.json`, `create-pet-image.json` and `get-pet-stats.json`?

Reply:
Updated/added new event files